impeller/blobcat/blob_writer.cc

```diff
@@ -31,7 +31,7 @@ bool BlobWriter::AddBlobAtPath(const std::string& std_path) {
     return false;
   }
 
-  if (path.extension() != ".gles") {
+  if (path.extension() != ".gles" && path.extension() != ".vkspv") {
```

Comment:
I may be missing context here but does this have any implication for the fragment shader API?

Reply:
No. This is just an intermediate in the gen directory. The Impeller FragmentProgram API will take these blobs, slap some metadata on it and embed it in the final binary.